extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -172,7 +172,12 @@ export class Bind {
           break;
         }
         const bindings = [];
-        for (const attr of el.attributes) {
+        const attrs = el.attributes;
+        for (let j = 0;; j++) {
```

Comment:
What's up with your weird for loops? 😛

You could just check `j < attrs.length`.